deploy/crds/operator.ibm.com_v1alpha1_metaoperatorcatalog_cr.yaml

```diff
@@ -20,3 +20,10 @@ spec:
     sourceNamespace: openshift-marketplace
     targetNamespaces:
       - etcd-operator
+  - name: ibm-cert-manager-operator
+    namespace: ibm-cs-certmanager
```

Comment:
can you set it to `ibm-cert-manager-operator`?

Reply:
Made and committed the change